cypress/integration/BackButtonMobile.js

```diff
@@ -0,0 +1,58 @@
+// @noflow
+/* global
+  Cypress,
+  cy
+*/
+
+describe('Back button mobile', () => {
+  beforeEach(() => {
+    cy.viewport('iphone-6');
+    cy.signIntoAccount();
+    cy.wait(2000);
```

Comment:
the reason why _wait_ is needed here, is because _back button for mobile_ is used just for logged in user, and for logged in user the root FAQs can change depending on status of booking, but initially the general root FAQ list is rendered

Reply:
then pls add explanation comment so this info is preserved in code

Reply:
or to avoid wait, change it to wait for FAQ to be loaded and then wait for "about Kiwi.com" category to disappear which means some after booking FAQ section category was loaded. I would expect 2 sec wait to be quite flaky.